sb_vision/tokens.py

```diff
@@ -51,57 +50,57 @@ def from_apriltag_detection(
     ) -> 'Token':
         """Construct a Token from an April Tag detection."""
 
-        pixel_coords = [Coordinate(*l) for l in apriltag_detection.p]
+        pixel_corners = [PixelCoordinate(*l) for l in apriltag_detection.p]
 
         # centre of marker: average the corners
-        pixel_centre = Coordinate(*np.average(pixel_coords, axis=0))
+        pixel_centre = PixelCoordinate(*np.average(pixel_corners, axis=0))
 
         marker_id = apriltag_detection.id
 
         # *************************************************************************
         # NOTE: IF YOU CHANGE THIS, THEN CHANGE ROBOT-API camera.py
         # *************************************************************************
+
+        # The pixel_corners value we expose has the first one in the bottom left.
+        # Apriltags returns the first one in the top left. thus we need to rotate
+        # the values by one to get the correct ordering.
+        offset_pixel_corners = [pixel_corners[3]] + pixel_corners[:3]
```

Comment:
It turns out that we don't appear to have documented the presence of the `pixel_corners` attribute, so I'm a bit happier about changing it if we felt we needed to do so.
Obviously we still need to understand what they mean and then document that either way.

Reply:
My OCD is killing me that this exists, however a team mentioned to me that they found this function, assuming they're using it, we can't silently change its behaviour just to remove one line of code

Reply:
I've carded up the task to undo this at https://trello.com/c/w9BzXPUV/81-change-pixelcorners-behaviour-to-be-the-same-as-apriltags so we can restore normality after this years game